app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+    # Атрибут класу для зберігання всіх створених екземплярів
+    people = {}
```

Comment:
The `people` dictionary is a class attribute that stores all instances of `Person`. This could lead to issues if `create_person_list` is called multiple times, as it will accumulate all instances across calls. Consider whether this is the intended behavior or if the dictionary should be reset or managed differently.